modelerfour/test/scenarios/a-playground/namer.yaml

```diff
@@ -199,35 +192,33 @@ schemas: !<!Schemas>
         precision: 64
         language: !<!Languages> 
           default:
-            name: typeForfield1
+            name: TypeForfield1
             description: ''
         protocol: !<!Protocols> {}
       serializedName: field1
       language: !<!Languages> 
         default:
           name: field1
           description: ''
-          originalName: field1
       protocol: !<!Protocols> {}
     - !<!Property> &ref_95
       schema: !<!NumberSchema> &ref_49
         type: integer
         precision: 64
         language: !<!Languages> 
           default:
-            name: typeForfield2
+            name: TypeForfield2
```

Comment:
Should all these renamed entities have the `originalName` field added with the unnormalized name?

Reply:
I don't know that there's a lot of value left with the original name. if a generator needs it I could.